template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApplication-CSharp/.template.config/template.json

```diff
@@ -52,6 +52,12 @@
       "datatype": "bool",
       "description": "If specified, skips the automatic restore of the project on create.",
       "defaultValue": "false"
+    },
```

Comment:
As agreed with @KathleenDollard, no changes to existing console template. This change will be done in Console App template (to be created shortly